### PR TITLE
Update hevc.json

### PR DESCRIPTION
**Pull request by sheik124**
Edge has supported HEVC since Version 77.
The UserVoice link previously here was:
1. Dated Oct 21, 2015!!!
2. Has been dead since 2019...
3. https://web.archive.org/web/20190624214549/https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer/suggestions/10311075-hevc-support

### CODE REVIEW

File: features-json/hevc.json [hunk 1]
```diff
@@ -485,7 +485,8 @@
   "notes_by_num":{
     "1":"Supported only for devices with [hardware support](https://answers.microsoft.com/en-us/insider/forum/insider_apps-insider_wmp/windows-10-hevc-playback-yes-or-no/3c1ab780-a6b2-4b77-ac0f-9faeefd4680d)",
     "2":"Reported to work in certain Android devices with hardware support",
-    "3":"Supported only on macOS High Sierra or later"
+    "3":"Supported only on macOS High Sierra or later",
+    "4":"Supported only for devices with [hardware support](https://techcommunity.microsoft.com/t5/discussions/updated-dev-channel-build-77-0-211-3-is-live/m-p/745801#M6548)
```
Comment (Schweinepriester) [inline features-json/hevc.json:489]:
```suggestion
    "4":"Supported only for devices with [hardware support](https://techcommunity.microsoft.com/t5/discussions/updated-dev-channel-build-77-0-211-3-is-live/m-p/745801#M6548)"
```

**Review by Schweinepriester — CHANGES_REQUESTED** [review]
Missing `"`, but other than that looking good!

Looks like I'm able to confirm as well, using Edge Version 101.0.1210.47 on Windows 10 21H2 on a Dell XPS 15 9510:

image

https://tests.caniuse.com/hevc shows the video as opposed to Chrome, which only gets the audio.

And I don't even have the [HEVC-extension](https://www.microsoft.com/store/productId/9NMZLZ57R3T7) installed.

There's a possible discussion if this warrants partial support or no support with the note.

**Comment by Fyrd** [comment]
I think based on the note partial support seems reasonable. Thanks!

**Comment by Fyrd** [comment]
I think based on the note partial support seems reasonable. Thanks!

**Pull request merged**